scripts/database.sh

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ASSUME_ROLE=${ENABLE_CROSSACCOUNT_ROLE:-0}
```

Comment:
Assuming` ENABLE_CROSSACCOUNT` value is  read via SSM parameter ?

Reply:
Yes, correct. Same as CROSSACCOUNT_ROLEARN